v1.17.2

sambacha opened the pull request:
Closes #

{{short description}}

#### Changelog

**New**

- {{new thing}}

**Changed**

- {{change thing}}

**Removed**

- {{removed thing}}

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/freight-trust/gatsby-theme-carbon-base?pullRequest=3) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/freight-trust/gatsby-theme-carbon-base?pullRequest=3) before we can accept your contribution.<br/>**0** out of **7** committers have signed the CLA.<br/><br/>:x: Ali Unwala<br/>:x: vpicone<br/>:x: sstrubberg<br/>:x: jeffchew<br/>:x: dakahn<br/>:x: kodiakhq[bot]<br/>:x: jnm2377<br/><hr/>**Ali Unwala** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/freight-trust/gatsby-theme-carbon-base?pullRequest=3) it.</sub>

eng618 committed:
chore: release 4.2.7 [skip ci],

eng618 committed:
chore: update lock file

dependabot[bot] committed:
build(deps): bump release-it from 17.10.0 to 18.1.2 (#1570)

Bumps [release-it](https://github.com/release-it/release-it) from 17.10.0 to 18.1.2.
- [Release notes](https://github.com/release-it/release-it/releases)
- [Changelog](https://github.com/release-it/release-it/blob/main/CHANGELOG.md)
- [Commits](https://github.com/release-it/release-it/compare/17.10.0...18.1.2)

---
updated-dependencies:
- dependency-name: release-it
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

eng618 committed:
build: remove unused dep

eng618 committed:
Revert "build(deps): bump release-it from 17.10.0 to 18.1.2 (#1570)"

This reverts commit 37abd24759b91dab635451ce78925b1f153031b8.

eng618 committed:
chore: release 4.2.8 [skip ci],

carbon-automation[bot] committed:
chore(release): update carbon deps (#1573)

Co-authored-by: 2nikhiltom <63502271+2nikhiltom@users.noreply.github.com>

eng618 committed:
chore: release 4.2.9 [skip ci],

guidari committed:
fix: fixed tab navigation in SideNav (#1567)

* fix: fixed tab navigation in SideNav

* Update packages/gatsby-theme-carbon/src/components/LeftNav/LeftNavItem.js

Co-authored-by: Alison Joseph <alisonejoseph@Gmail.com>

* Update packages/gatsby-theme-carbon/src/components/LeftNav/ResourceLinks.js

Co-authored-by: Alison Joseph <alisonejoseph@Gmail.com>

* fix: fixed hamburger menu with infinite looping

---------

Co-authored-by: Alison Joseph <alisonejoseph@Gmail.com>
Co-authored-by: Alison Joseph <alison.joseph@us.ibm.com>

eng618 committed:
chore: release 4.2.10 [skip ci],

carbon-automation[bot] committed:
chore(release): update carbon deps (#1574)

Co-authored-by: ariellalgilmore <20210594+ariellalgilmore@users.noreply.github.com>

eng618 committed:
chore: release 4.2.11 [skip ci],

tay1orjones committed:
chore: add .nvmrc for node versioning

carbon-automation[bot] committed:
chore(release): update carbon deps (#1575)

Co-authored-by: ariellalgilmore <20210594+ariellalgilmore@users.noreply.github.com>

eng618 committed:
chore: release 4.2.12 [skip ci],

preetibansalui committed:
fix: allow multiple medium accounts (#1578)

* fix: allow multiple medium accounts

* fix: lock file error

* fix: update documents

* fix: prettier format

eng618 committed:
chore: release 4.2.13 [skip ci],

carbon-automation[bot] committed:
chore(release): update carbon deps (#1577)

Co-authored-by: 2nikhiltom <63502271+2nikhiltom@users.noreply.github.com>

eng618 committed:
chore: release 4.2.14 [skip ci],

dependabot[bot] committed:
build(deps-dev): bump eslint-config-prettier from 10.0.1 to 10.1.1 (#1581)

Bumps [eslint-config-prettier](https://github.com/prettier/eslint-config-prettier) from 10.0.1 to 10.1.1.
- [Release notes](https://github.com/prettier/eslint-config-prettier/releases)
- [Changelog](https://github.com/prettier/eslint-config-prettier/blob/main/CHANGELOG.md)
- [Commits](https://github.com/prettier/eslint-config-prettier/compare/v10.0.1...v10.1.1)

---
updated-dependencies:
- dependency-name: eslint-config-prettier
  dependency-version: 10.1.1
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

eng618 committed:
chore: release 4.2.15 [skip ci],

annawen1 committed:
chore: update validate workflow ubuntu version (#1586)

* chore: update validate workflow ubuntu version

* chore: update yarn.lock file

eng618 committed:
chore: release 4.2.16 [skip ci],

carbon-automation[bot] committed:
chore(release): update carbon deps (#1585)

Co-authored-by: annawen1 <54281166+annawen1@users.noreply.github.com>

eng618 committed:
chore: release 4.2.17 [skip ci],

dependabot[bot] committed:
build(deps): bump the npm_and_yarn group with 3 updates (#1576)

Bumps the npm_and_yarn group with 3 updates: [@babel/helpers](https://github.com/babel/babel/tree/HEAD/packages/babel-helpers), [@babel/runtime](https://github.com/babel/babel/tree/HEAD/packages/babel-runtime) and [axios](https://github.com/axios/axios).


Updates `@babel/helpers` from 7.26.0 to 7.26.10
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/main/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/commits/v7.26.10/packages/babel-helpers)

Updates `@babel/runtime` from 7.26.0 to 7.26.10
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/main/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/commits/v7.26.10/packages/babel-runtime)

Updates `axios` from 1.7.9 to 1.8.3
- [Release notes](https://github.com/axios/axios/releases)
- [Changelog](https://github.com/axios/axios/blob/v1.x/CHANGELOG.md)
- [Commits](https://github.com/axios/axios/compare/v1.7.9...v1.8.3)

---
updated-dependencies:
- dependency-name: "@babel/helpers"
  dependency-type: indirect
  dependency-group: npm_and_yarn
- dependency-name: "@babel/runtime"
  dependency-type: indirect
  dependency-group: npm_and_yarn
- dependency-name: axios
  dependency-type: indirect
  dependency-group: npm_and_yarn
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

eng618 committed:
chore: release 4.2.18 [skip ci],

eng618 committed:
build(deps): bump minor and below (#1587)

eng618 committed:
chore: release 4.2.19 [skip ci],

dependabot[bot] committed:
build(deps): bump tar-fs from 2.1.1 to 2.1.2 in the npm_and_yarn group (#1588)

Bumps the npm_and_yarn group with 1 update: [tar-fs](https://github.com/mafintosh/tar-fs).


Updates `tar-fs` from 2.1.1 to 2.1.2
- [Commits](https://github.com/mafintosh/tar-fs/compare/v2.1.1...v2.1.2)

---
updated-dependencies:
- dependency-name: tar-fs
  dependency-version: 2.1.2
  dependency-type: indirect
  dependency-group: npm_and_yarn
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

eng618 committed:
chore: release 4.2.20 [skip ci],

dependabot[bot] committed:
build(deps): bump the npm_and_yarn group with 3 updates (#1589)

Bumps the npm_and_yarn group with 3 updates: [@octokit/endpoint](https://github.com/octokit/endpoint.js), [@octokit/request-error](https://github.com/octokit/request-error.js) and [@octokit/request](https://github.com/octokit/request.js).


Updates `@octokit/endpoint` from 9.0.5 to 9.0.6
- [Release notes](https://github.com/octokit/endpoint.js/releases)
- [Commits](https://github.com/octokit/endpoint.js/compare/v9.0.5...v9.0.6)

Updates `@octokit/request-error` from 5.1.0 to 5.1.1
- [Release notes](https://github.com/octokit/request-error.js/releases)
- [Commits](https://github.com/octokit/request-error.js/compare/v5.1.0...v5.1.1)

Updates `@octokit/request` from 8.4.0 to 8.4.1
- [Release notes](https://github.com/octokit/request.js/releases)
- [Commits](https://github.com/octokit/request.js/compare/v8.4.0...v8.4.1)

---
updated-dependencies:
- dependency-name: "@octokit/endpoint"
  dependency-version: 9.0.6
  dependency-type: indirect
  dependency-group: npm_and_yarn
- dependency-name: "@octokit/request-error"
  dependency-version: 5.1.1
  dependency-type: indirect
  dependency-group: npm_and_yarn
- dependency-name: "@octokit/request"
  dependency-version: 8.4.1
  dependency-type: indirect
  dependency-group: npm_and_yarn
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

eng618 committed:
chore: release 4.2.21 [skip ci],

eng618 committed:
ci: update to latest caching practices

eng618 committed:
fix lockfile

eng618 committed:
feat(release): batch release of all packages weekly (#1590)

* feat(release): batch release of all packages weekly

Also enhance release workflow to check for new commits and validate status before release,
so that we do not release if there are no new commits or if the validate workflow fails.

* add action permissions

eng618 committed:
ci: ensure release can only be ran on main

eng618 committed:
ci: update logic for manual trigger to only be allowed by authorized users

eng618 committed:
feat(gh-pages): add GitHub Actions workflow for deploying Gatsby site to GitHub Pages (#1591)

* feat(gh-pages): add GitHub Actions workflow for deploying Gatsby site to GitHub Pages

* cleanup

eng618 committed:
ci: fix publix build path

eng618 committed:
ci: Update gh-pages workflow and add CNAME file

eng618 committed:
fix: remove unused prefix

eng618 committed:
ci: fix syntax error

eng618 committed:
ci: fix syntax error

eng618 committed:
ci: debug release

eng618 committed:
ci: degugging release workflow

eng618 committed:
ci: degugging release workflow

eng618 committed:
ci: update permissions in workflows for release and validation

eng618 committed:
ci: degugging release workflow

eng618 committed:
ci: degugging release workflow

eng618 committed:
ci: degugging release workflow

eng618 committed:
ci: degugging release workflow

eng618 committed:
ci: degugging release workflow

eng618 committed:
ci: degugging release workflow

eng618 committed:
chore: release 4.3.0 [skip ci],

eng618 committed:
ci: simplify the scheduled release workflow

eng618 committed:
ci: simplify the manual release workflow

eng618 committed:
fix: update lock

eng618 committed:
fix: add pathPrefix to gatsby-config

eng618 committed:
chore: release 4.3.1 [skip ci],

alisonjoseph committed:
Update README.md (#1592)

alisonjoseph committed:
chore: deploy (#1593)

eng618 committed:
chore: release 4.3.2 [skip ci],

alisonjoseph committed:
chore: add cname for deploy (#1594)

alisonjoseph committed:
chore: remove pathprefix (#1595)

alisonjoseph committed:
chore: copy images deployment (#1596)

alisonjoseph committed:
Update gh-pages.yml (#1597)

alisonjoseph committed:
chore: build:clean for deployment (#1598)

alisonjoseph committed:
chore: remove vercel (#1599)

eng618 committed:
chore: release 4.3.3 [skip ci],

dependabot[bot] committed:
build(deps): bump base-x from 3.0.10 to 3.0.11 in the npm_and_yarn group (#1600)

Bumps the npm_and_yarn group with 1 update: [base-x](https://github.com/cryptocoinjs/base-x).


Updates `base-x` from 3.0.10 to 3.0.11
- [Commits](https://github.com/cryptocoinjs/base-x/compare/v3.0.10...v3.0.11)

---
updated-dependencies:
- dependency-name: base-x
  dependency-version: 3.0.11
  dependency-type: indirect
  dependency-group: npm_and_yarn
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump @babel/preset-react from 7.26.3 to 7.27.1 (#1601)

Bumps [@babel/preset-react](https://github.com/babel/babel/tree/HEAD/packages/babel-preset-react) from 7.26.3 to 7.27.1.
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/main/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/commits/v7.27.1/packages/babel-preset-react)

---
updated-dependencies:
- dependency-name: "@babel/preset-react"
  dependency-version: 7.27.1
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump stylelint from 16.19.0 to 16.19.1 (#1603)

Bumps [stylelint](https://github.com/stylelint/stylelint) from 16.19.0 to 16.19.1.
- [Release notes](https://github.com/stylelint/stylelint/releases)
- [Changelog](https://github.com/stylelint/stylelint/blob/main/CHANGELOG.md)
- [Commits](https://github.com/stylelint/stylelint/compare/16.19.0...16.19.1)

---
updated-dependencies:
- dependency-name: stylelint
  dependency-version: 16.19.1
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump @babel/core from 7.26.10 to 7.27.1 (#1602)

Bumps [@babel/core](https://github.com/babel/babel/tree/HEAD/packages/babel-core) from 7.26.10 to 7.27.1.
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/main/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/commits/v7.27.1/packages/babel-core)

---
updated-dependencies:
- dependency-name: "@babel/core"
  dependency-version: 7.27.1
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

carbon-automation[bot] committed:
chore(release): update carbon deps (#1605)

Co-authored-by: annawen1 <54281166+annawen1@users.noreply.github.com>

eng618 committed:
chore: release 4.3.4 [skip ci],

dependabot[bot] committed:
build(deps): bump multer in the npm_and_yarn group (#1606)

Bumps the npm_and_yarn group with 1 update: [multer](https://github.com/expressjs/multer).


Updates `multer` from 1.4.5-lts.1 to 1.4.5-lts.2
- [Release notes](https://github.com/expressjs/multer/releases)
- [Changelog](https://github.com/expressjs/multer/blob/v1.4.5-lts.2/CHANGELOG.md)
- [Commits](https://github.com/expressjs/multer/compare/v1.4.5-lts.1...v1.4.5-lts.2)

---
updated-dependencies:
- dependency-name: multer
  dependency-version: 1.4.5-lts.2
  dependency-type: indirect
  dependency-group: npm_and_yarn
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

carbon-automation[bot] committed:
chore(release): update carbon deps (#1607)

Co-authored-by: kennylam <909118+kennylam@users.noreply.github.com>

eng618 committed:
chore: release 4.3.5 [skip ci],

dependabot[bot] committed:
build(deps): bump tar-fs from 2.1.2 to 2.1.3 in the npm_and_yarn group (#1608)

Bumps the npm_and_yarn group with 1 update: [tar-fs](https://github.com/mafintosh/tar-fs).


Updates `tar-fs` from 2.1.2 to 2.1.3
- [Commits](https://github.com/mafintosh/tar-fs/commits)

---
updated-dependencies:
- dependency-name: tar-fs
  dependency-version: 2.1.3
  dependency-type: indirect
  dependency-group: npm_and_yarn
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

jeffchew committed:
chore(vimeo): remove final vimeo videos (#1609)

eng618 committed:
chore: release 4.3.6 [skip ci],

dependabot[bot] committed:
build(deps): bump lint-staged from 15.5.1 to 16.1.0 (#1612)

Bumps [lint-staged](https://github.com/lint-staged/lint-staged) from 15.5.1 to 16.1.0.
- [Release notes](https://github.com/lint-staged/lint-staged/releases)
- [Changelog](https://github.com/lint-staged/lint-staged/blob/main/CHANGELOG.md)
- [Commits](https://github.com/lint-staged/lint-staged/compare/v15.5.1...v16.1.0)

---
updated-dependencies:
- dependency-name: lint-staged
  dependency-version: 16.1.0
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

eng618 committed:
chore: npx update-browserslist-db@latest

dependabot[bot] committed:
build(deps-dev): bump eslint-plugin-html from 8.1.2 to 8.1.3 (#1611)

Bumps [eslint-plugin-html](https://github.com/BenoitZugmeyer/eslint-plugin-html) from 8.1.2 to 8.1.3.
- [Changelog](https://github.com/BenoitZugmeyer/eslint-plugin-html/blob/main/CHANGELOG.md)
- [Commits](https://github.com/BenoitZugmeyer/eslint-plugin-html/compare/v8.1.2...v8.1.3)

---
updated-dependencies:
- dependency-name: eslint-plugin-html
  dependency-version: 8.1.3
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump stylelint from 16.19.1 to 16.20.0 (#1610)

Bumps [stylelint](https://github.com/stylelint/stylelint) from 16.19.1 to 16.20.0.
- [Release notes](https://github.com/stylelint/stylelint/releases)
- [Changelog](https://github.com/stylelint/stylelint/blob/main/CHANGELOG.md)
- [Commits](https://github.com/stylelint/stylelint/compare/16.19.1...16.20.0)

---
updated-dependencies:
- dependency-name: stylelint
  dependency-version: 16.20.0
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump @babel/eslint-parser from 7.27.0 to 7.27.1 (#1613)

Bumps [@babel/eslint-parser](https://github.com/babel/babel/tree/HEAD/eslint/babel-eslint-parser) from 7.27.0 to 7.27.1.
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/main/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/commits/v7.27.1/eslint/babel-eslint-parser)

---
updated-dependencies:
- dependency-name: "@babel/eslint-parser"
  dependency-version: 7.27.1
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump eslint-plugin-prettier from 5.2.6 to 5.4.1 (#1614)

Bumps [eslint-plugin-prettier](https://github.com/prettier/eslint-plugin-prettier) from 5.2.6 to 5.4.1.
- [Release notes](https://github.com/prettier/eslint-plugin-prettier/releases)
- [Changelog](https://github.com/prettier/eslint-plugin-prettier/blob/main/CHANGELOG.md)
- [Commits](https://github.com/prettier/eslint-plugin-prettier/compare/v5.2.6...v5.4.1)

---
updated-dependencies:
- dependency-name: eslint-plugin-prettier
  dependency-version: 5.4.1
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

carbon-automation[bot] committed:
chore(release): update carbon deps (#1615)

Co-authored-by: alisonjoseph <2753488+alisonjoseph@users.noreply.github.com>

eng618 committed:
chore: release 4.3.7 [skip ci],

dependabot[bot] committed:
build(deps): bump brace-expansion in the npm_and_yarn group (#1616)

Bumps the npm_and_yarn group with 1 update: [brace-expansion](https://github.com/juliangruber/brace-expansion).


Updates `brace-expansion` from 1.1.11 to 1.1.12
- [Release notes](https://github.com/juliangruber/brace-expansion/releases)
- [Commits](https://github.com/juliangruber/brace-expansion/compare/1.1.11...v1.1.12)

---
updated-dependencies:
- dependency-name: brace-expansion
  dependency-version: 1.1.12
  dependency-type: indirect
  dependency-group: npm_and_yarn
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

eng618 committed:
chore: release 4.3.8 [skip ci],

carbon-automation[bot] committed:
chore(release): update carbon deps (#1617)

Co-authored-by: riddhybansal <41935566+riddhybansal@users.noreply.github.com>

carbon-automation[bot] committed:
chore(release): update carbon deps (#1623)

Co-authored-by: tay1orjones <3360588+tay1orjones@users.noreply.github.com>

tay1orjones committed:
Update release.yml

eng618 committed:
chore: release 4.3.9 [skip ci],

carbon-automation[bot] committed:
chore(release): update carbon deps (#1624)

Co-authored-by: 2nikhiltom <63502271+2nikhiltom@users.noreply.github.com>

eng618 committed:
chore: release 4.3.10 [skip ci],

carbon-automation[bot] committed:
chore(release): update carbon deps (#1626)

Co-authored-by: riddhybansal <41935566+riddhybansal@users.noreply.github.com>

eng618 committed:
chore: update CODEOWNERS and format dependabot.yml for consistency

eng618 committed:
chore: update CODEOWNERS to assign ownership to @eng618

eng618 committed:
chore: update Yarn version to 4.9.2 and adjust package manager in package.json

- Updated .yarnrc.yml to use Yarn 4.9.2
- Changed packageManager field in package.json to reflect Yarn 4.9.2
- Updated yarn.lock to reflect changes in TypeScript resolution and checksum

eng618 committed:
fix: lock file

eng618 committed:
build(deps): bump all minor and below (#1628)

* build(deps): bump all minor and below

* fix: formatting

* also bump prettier-config-carbon

eng618 committed:
chore: release 4.3.11 [skip ci],

carbon-automation[bot] committed:
chore(release): update carbon deps (#1629)

Co-authored-by: riddhybansal <41935566+riddhybansal@users.noreply.github.com>

eng618 committed:
chore: release 4.3.12 [skip ci],

carbon-automation[bot] committed:
chore(release): update carbon deps (#1630)

Co-authored-by: ariellalgilmore <20210594+ariellalgilmore@users.noreply.github.com>

eng618 committed:
chore: release 4.3.13 [skip ci],

carbon-automation[bot] committed:
chore(release): update carbon deps (#1635)

Co-authored-by: tay1orjones <3360588+tay1orjones@users.noreply.github.com>

eng618 committed:
chore: release 4.3.14 [skip ci],

carbon-automation[bot] committed:
chore(release): update carbon deps (#1636)

Co-authored-by: tay1orjones <3360588+tay1orjones@users.noreply.github.com>

eng618 committed:
chore: release 4.3.15 [skip ci],

tay1orjones committed:
Update CODEOWNERS

dependabot[bot] committed:
build(deps): bump gatsby from 5.14.5 to 5.15.0 (#1634)

Bumps [gatsby](https://github.com/gatsbyjs/gatsby) from 5.14.5 to 5.15.0.
- [Release notes](https://github.com/gatsbyjs/gatsby/releases)
- [Changelog](https://github.com/gatsbyjs/gatsby/blob/master/CHANGELOG.md)
- [Commits](https://github.com/gatsbyjs/gatsby/compare/gatsby@5.14.5...gatsby@5.15.0)

---
updated-dependencies:
- dependency-name: gatsby
  dependency-version: 5.15.0
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Co-authored-by: Alison Joseph <alisonejoseph@Gmail.com>

eng618 committed:
chore: release 4.3.16 [skip ci],

eng618 committed:
build: update tooling

dependabot[bot] committed:
build(deps): bump actions/setup-node from 4 to 5 (#1638)

Bumps [actions/setup-node](https://github.com/actions/setup-node) from 4 to 5.
- [Release notes](https://github.com/actions/setup-node/releases)
- [Commits](https://github.com/actions/setup-node/compare/v4...v5)

---
updated-dependencies:
- dependency-name: actions/setup-node
  dependency-version: '5'
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

eng618 committed:
chore: update GitHub Actions workflows to use actions/checkout@v5 and Node.js 22

dependabot[bot] committed:
build(deps): bump actions/github-script from 7 to 8 (#1639)

Bumps [actions/github-script](https://github.com/actions/github-script) from 7 to 8.
- [Release notes](https://github.com/actions/github-script/releases)
- [Commits](https://github.com/actions/github-script/compare/v7...v8)

---
updated-dependencies:
- dependency-name: actions/github-script
  dependency-version: '8'
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump actions/upload-pages-artifact from 3 to 4 (#1633)

Bumps [actions/upload-pages-artifact](https://github.com/actions/upload-pages-artifact) from 3 to 4.
- [Release notes](https://github.com/actions/upload-pages-artifact/releases)
- [Commits](https://github.com/actions/upload-pages-artifact/compare/v3...v4)

---
updated-dependencies:
- dependency-name: actions/upload-pages-artifact
  dependency-version: '4'
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

carbon-automation[bot] committed:
chore(release): update carbon deps (#1641)

Co-authored-by: maradwan26 <132728978+maradwan26@users.noreply.github.com>

eng618 committed:
chore: release 4.3.17 [skip ci],

tay1orjones committed:
chore(ci): update allowed actors for running release workflow

dependabot[bot] committed:
build(deps): bump lerna from 8.2.4 to 9.0.0 (#1640)

Bumps [lerna](https://github.com/lerna/lerna/tree/HEAD/packages/lerna) from 8.2.4 to 9.0.0.
- [Release notes](https://github.com/lerna/lerna/releases)
- [Changelog](https://github.com/lerna/lerna/blob/main/packages/lerna/CHANGELOG.md)
- [Commits](https://github.com/lerna/lerna/commits/v9.0.0/packages/lerna)

---
updated-dependencies:
- dependency-name: lerna
  dependency-version: 9.0.0
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

eng618 committed:
chore: release 4.3.18 [skip ci],

carbon-automation[bot] committed:
chore(release): update carbon deps (#1642)

Co-authored-by: tay1orjones <3360588+tay1orjones@users.noreply.github.com>

eng618 committed:
chore: release 4.3.19 [skip ci],

carbon-automation[bot] committed:
chore(release): update carbon deps (#1645)

Co-authored-by: Gururajj77 <89023023+Gururajj77@users.noreply.github.com>

alisonjoseph committed:
Remove Racial Equity in Design link from Switcher (#1646)

dependabot[bot] committed:
build(deps): bump actions/setup-node from 4 to 6 (#1643)

dependabot[bot] committed:
build(deps): bump @release-it/conventional-changelog (#1621)

Bumps [@release-it/conventional-changelog](https://github.com/release-it/conventional-changelog) from 8.0.2 to 10.0.1.
- [Release notes](https://github.com/release-it/conventional-changelog/releases)
- [Commits](https://github.com/release-it/conventional-changelog/compare/8.0.2...10.0.1)

---
updated-dependencies:
- dependency-name: "@release-it/conventional-changelog"
  dependency-version: 10.0.1
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump stylelint from 16.23.0 to 16.25.0 (#1644)

Bumps [stylelint](https://github.com/stylelint/stylelint) from 16.23.0 to 16.25.0.
- [Release notes](https://github.com/stylelint/stylelint/releases)
- [Changelog](https://github.com/stylelint/stylelint/blob/main/CHANGELOG.md)
- [Commits](https://github.com/stylelint/stylelint/compare/16.23.0...16.25.0)

---
updated-dependencies:
- dependency-name: stylelint
  dependency-version: 16.25.0
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

eng618 committed:
chore: release 4.3.20 [skip ci],

eng618 committed:
chore: update gatsby-theme-carbon to v4.3.20 and enable merge group for CI workflow

- Bump gatsby-theme-carbon from ^4.3.19 to ^4.3.20 in yarn.lock for latest features/fixes.
- Add merge_group trigger to .github/workflows/gh-pages.yml to support GitHub's merge queues, ensuring deployment runs on merge group pushes.

eng618 committed:
chore(ci): refactor release workflow to update lockfile via release-it hooks

Remove the separate 'update-lockfile' job from GitHub Actions and add release-it hooks to handle yarn.lock updates during the release commit, streamlining the process and avoiding extra commits.

dependabot[bot] committed:
build(deps): bump @release-it-plugins/workspaces from 4.2.0 to 5.0.3 (#1619)

Bumps [@release-it-plugins/workspaces](https://github.com/release-it-plugins/workspaces) from 4.2.0 to 5.0.3.
- [Release notes](https://github.com/release-it-plugins/workspaces/releases)
- [Changelog](https://github.com/release-it-plugins/workspaces/blob/master/CHANGELOG.md)
- [Commits](https://github.com/release-it-plugins/workspaces/compare/v4.2.0...v5.0.3)

---
updated-dependencies:
- dependency-name: "@release-it-plugins/workspaces"
  dependency-version: 5.0.3
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump form-data in the npm_and_yarn group (#1625)

Bumps the npm_and_yarn group with 1 update: [form-data](https://github.com/form-data/form-data).


Updates `form-data` from 4.0.1 to 4.0.4
- [Release notes](https://github.com/form-data/form-data/releases)
- [Changelog](https://github.com/form-data/form-data/blob/master/CHANGELOG.md)
- [Commits](https://github.com/form-data/form-data/compare/v4.0.1...v4.0.4)

---
updated-dependencies:
- dependency-name: form-data
  dependency-version: 4.0.4
  dependency-type: indirect
  dependency-group: npm_and_yarn
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump gatsby-plugin-catch-links from 5.14.0 to 5.15.0 (#1637)

Bumps [gatsby-plugin-catch-links](https://github.com/gatsbyjs/gatsby/tree/HEAD/packages/gatsby-plugin-catch-links) from 5.14.0 to 5.15.0.
- [Release notes](https://github.com/gatsbyjs/gatsby/releases)
- [Changelog](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-plugin-catch-links/CHANGELOG.md)
- [Commits](https://github.com/gatsbyjs/gatsby/commits/gatsby-plugin-catch-links@5.15.0/packages/gatsby-plugin-catch-links)

---
updated-dependencies:
- dependency-name: gatsby-plugin-catch-links
  dependency-version: 5.15.0
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Co-authored-by: Nikhil Tomar <63502271+2nikhiltom@users.noreply.github.com>
Co-authored-by: Eric Garcia <eng618@garciaericn.com>

eng618 committed:
chore(deps): update Babel, MDX, Lerna, and other dependencies to late… (#1647)

* chore(deps): update Babel, MDX, Lerna, and other dependencies to latest patch versions

Updated @babel/core from ^7.28.0 to ^7.28.5, @babel/eslint-parser from ^7.28.0 to ^7.28.5, lerna from ^9.0.0 to ^9.0.1, eslint-plugin-prettier from ^5.5.3 to ^5.5.4, and @mdx-js/mdx and @mdx-js/react from ^3.1.0 to ^3.1.1 for bug fixes and improvements. Yarn lock updated accordingly.

* chore(deps): update dependencies to latest versions

- Updated Babel, ESLint, lint-staged, stylelint, and related devDependencies in root package.json
- Updated Gatsby plugins from v5.14.x to v5.15.0 and other dependencies in example and gatsby-theme-carbon packages
- Updated @carbon/icons-react and @vimeo/player to newer versions for better compatibility and features

dependabot[bot] committed:
build(deps): bump the npm_and_yarn group across 1 directory with 3 updates (#1648)

carbon-automation[bot] committed:
chore(release): update carbon deps (#1649)

alisonjoseph committed:
chore: update lockfile (#1650)

eng618 committed:
chore: configure independent versioning and update tooling documentation

- Updated lerna.json to use independent versioning for monorepo packages
- Modified package.json to fix husky commit-msg hook for compatibility
- Enhanced docs with details on package manager versioning, pre-commit hooks (husky/lint-staged/commitlint), commitment fixes, and release processes
- Fixed typos and image links in dependency management docs for better clarity<|reserved_21|>This commit updates project configuration for independent versioning in the monorepo and enhances related tooling documentation, ensuring consistency and automated quality checks across environments. It removes the fixed version constraint in lerna.json to allow packages to version independently, fixes the husky commit-msg hook in package.json for improved compatibility, and expands docs with clarifications on package manager fields, pre-commit hooks (using husky, lint-staged, and commitlint), commit amendment processes, and prerelease workflows. Minor fixes like typo corrections and link updates are included for accuracy and usability.

eng618 committed:
build: bump deps

eng618 committed:
chore: downgrade @release-it-plugins/workspaces to v4.2.0 and update dependencies

eng618 committed:
deps: update `@release-it-plugins/workspaces` and `release-it` dependencies.

eng618 committed:
chore: Add conventional changelog dev dependency and configure npm publish registry.

eng618 committed:
chore: add --no-immutable flag to yarn install in after:bump hook

eng618 committed:
chore: release 4.3.21 [skip ci]

dependabot[bot] committed:
build(deps): bump actions/checkout from 5 to 6 (#1651)

Bumps [actions/checkout](https://github.com/actions/checkout) from 5 to 6.
- [Release notes](https://github.com/actions/checkout/releases)
- [Changelog](https://github.com/actions/checkout/blob/main/CHANGELOG.md)
- [Commits](https://github.com/actions/checkout/compare/v5...v6)

---
updated-dependencies:
- dependency-name: actions/checkout
  dependency-version: '6'
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump express in the npm_and_yarn group across 1 directory (#1657)

Bumps the npm_and_yarn group with 1 update in the / directory: [express](https://github.com/expressjs/express).


Updates `express` from 4.21.2 to 4.22.1
- [Release notes](https://github.com/expressjs/express/releases)
- [Changelog](https://github.com/expressjs/express/blob/v4.22.1/History.md)
- [Commits](https://github.com/expressjs/express/compare/4.21.2...v4.22.1)

---
updated-dependencies:
- dependency-name: express
  dependency-version: 4.22.1
  dependency-type: indirect
  dependency-group: npm_and_yarn
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

carbon-automation[bot] committed:
chore(release): update carbon deps (#1658)

Co-authored-by: riddhybansal <41935566+riddhybansal@users.noreply.github.com>

eng618 committed:
chore: release 4.3.22 [skip ci]

eng618 committed:
fix: Resolve SSR build issues by nullifying smooth-scroll during SSR and guarding window object access in hooks.

eng618 committed:
config: Enable modern Sass API and silence legacy JS API deprecation.

eng618 committed:
chore: release 4.3.23 [skip ci]

dependabot[bot] committed:
build(deps): bump focus-trap-react from 10.3.1 to 11.0.4 (#1655)

Bumps [focus-trap-react](https://github.com/focus-trap/focus-trap-react) from 10.3.1 to 11.0.4.
- [Release notes](https://github.com/focus-trap/focus-trap-react/releases)
- [Changelog](https://github.com/focus-trap/focus-trap-react/blob/master/CHANGELOG.md)
- [Commits](https://github.com/focus-trap/focus-trap-react/compare/v10.3.1...v11.0.4)

---
updated-dependencies:
- dependency-name: focus-trap-react
  dependency-version: 11.0.4
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump @commitlint/cli from 18.6.1 to 20.1.0 (#1654)

Bumps [@commitlint/cli](https://github.com/conventional-changelog/commitlint/tree/HEAD/@commitlint/cli) from 18.6.1 to 20.1.0.
- [Release notes](https://github.com/conventional-changelog/commitlint/releases)
- [Changelog](https://github.com/conventional-changelog/commitlint/blob/master/@commitlint/cli/CHANGELOG.md)
- [Commits](https://github.com/conventional-changelog/commitlint/commits/v20.1.0/@commitlint/cli)

---
updated-dependencies:
- dependency-name: "@commitlint/cli"
  dependency-version: 20.1.0
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump @commitlint/config-conventional from 18.6.3 to 20.0.0 (#1653)

Bumps [@commitlint/config-conventional](https://github.com/conventional-changelog/commitlint/tree/HEAD/@commitlint/config-conventional) from 18.6.3 to 20.0.0.
- [Release notes](https://github.com/conventional-changelog/commitlint/releases)
- [Changelog](https://github.com/conventional-changelog/commitlint/blob/master/@commitlint/config-conventional/CHANGELOG.md)
- [Commits](https://github.com/conventional-changelog/commitlint/commits/v20.0.0/@commitlint/config-conventional)

---
updated-dependencies:
- dependency-name: "@commitlint/config-conventional"
  dependency-version: 20.0.0
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump eslint-plugin-react-hooks from 5.2.0 to 7.0.1 (#1652)

Bumps [eslint-plugin-react-hooks](https://github.com/facebook/react/tree/HEAD/packages/eslint-plugin-react-hooks) from 5.2.0 to 7.0.1.
- [Release notes](https://github.com/facebook/react/releases)
- [Changelog](https://github.com/facebook/react/blob/main/packages/eslint-plugin-react-hooks/CHANGELOG.md)
- [Commits](https://github.com/facebook/react/commits/HEAD/packages/eslint-plugin-react-hooks)

---
updated-dependencies:
- dependency-name: eslint-plugin-react-hooks
  dependency-version: 7.0.1
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

eng618 committed:
chore: Silence additional Sass deprecation warnings for `global-builtin` and `if-function`.

eng618 committed:
style: reformat sassOptions.silenceDeprecations array for improved readability.

eng618 committed:
chore: release 4.3.24 [skip ci]

maradwan26 committed:
fix(globalsearch): fix page crash when typing : or ^ (#1659)

maradwan26 committed:
fix(globalsearch): tweak regex to properly replace (#1660)

eng618 committed:
chore: release 4.3.25 [skip ci]

carbon-automation[bot] committed:
chore(release): update carbon deps (#1661)

Co-authored-by: riddhybansal <41935566+riddhybansal@users.noreply.github.com>

eng618 committed:
chore: release 4.3.26 [skip ci]

dependabot[bot] committed:
build(deps): bump actions/cache from 4 to 5 (#1662)

Bumps [actions/cache](https://github.com/actions/cache) from 4 to 5.
- [Release notes](https://github.com/actions/cache/releases)
- [Changelog](https://github.com/actions/cache/blob/main/RELEASES.md)
- [Commits](https://github.com/actions/cache/compare/v4...v5)

---
updated-dependencies:
- dependency-name: actions/cache
  dependency-version: '5'
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

eng618 committed:
chore: release 4.3.27 [skip ci]

dependabot[bot] committed:
build(deps-dev): bump eslint from 9.39.1 to 9.39.2 (#1665)

Bumps [eslint](https://github.com/eslint/eslint) from 9.39.1 to 9.39.2.
- [Release notes](https://github.com/eslint/eslint/releases)
- [Commits](https://github.com/eslint/eslint/compare/v9.39.1...v9.39.2)

---
updated-dependencies:
- dependency-name: eslint
  dependency-version: 9.39.2
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump @eslint/js from 9.39.1 to 9.39.2 (#1663)

Bumps [@eslint/js](https://github.com/eslint/eslint/tree/HEAD/packages/js) from 9.39.1 to 9.39.2.
- [Release notes](https://github.com/eslint/eslint/releases)
- [Commits](https://github.com/eslint/eslint/commits/v9.39.2/packages/js)

---
updated-dependencies:
- dependency-name: "@eslint/js"
  dependency-version: 9.39.2
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump prettier from 3.6.2 to 3.7.4 (#1666)

Bumps [prettier](https://github.com/prettier/prettier) from 3.6.2 to 3.7.4.
- [Release notes](https://github.com/prettier/prettier/releases)
- [Changelog](https://github.com/prettier/prettier/blob/main/CHANGELOG.md)
- [Commits](https://github.com/prettier/prettier/compare/3.6.2...3.7.4)

---
updated-dependencies:
- dependency-name: prettier
  dependency-version: 3.7.4
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump release-it from 19.0.6 to 19.2.2 (#1667)

Bumps [release-it](https://github.com/release-it/release-it) from 19.0.6 to 19.2.2.
- [Release notes](https://github.com/release-it/release-it/releases)
- [Changelog](https://github.com/release-it/release-it/blob/main/CHANGELOG.md)
- [Commits](https://github.com/release-it/release-it/compare/19.0.6...19.2.2)

---
updated-dependencies:
- dependency-name: release-it
  dependency-version: 19.2.2
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

carbon-automation[bot] committed:
chore(release): update carbon deps (#1669)

Co-authored-by: kennylam <909118+kennylam@users.noreply.github.com>

eng618 committed:
chore: release 4.3.28 [skip ci]

carbon-automation[bot] committed:
chore(release): update carbon deps (#1670)

eng618 committed:
chore: release 4.3.29 [skip ci]

dependabot[bot] committed:
build(deps): bump diff in the npm_and_yarn group across 1 directory (#1671)

Bumps the npm_and_yarn group with 1 update in the / directory: [diff](https://github.com/kpdecker/jsdiff).


Updates `diff` from 5.2.0 to 5.2.2
- [Changelog](https://github.com/kpdecker/jsdiff/blob/master/release-notes.md)
- [Commits](https://github.com/kpdecker/jsdiff/compare/v5.2.0...v5.2.2)

---
updated-dependencies:
- dependency-name: diff
  dependency-version: 5.2.2
  dependency-type: indirect
  dependency-group: npm_and_yarn
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

eng618 committed:
chore: release 4.3.30 [skip ci]

dependabot[bot] committed:
build(deps): bump lodash in the npm_and_yarn group across 1 directory (#1672)

carbon-automation[bot] committed:
chore(release): update carbon deps (#1673)

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

eng618 committed:
chore: release 4.3.31 [skip ci]

eng618 committed:
ci: enable trusted publisher for npm

remove explicit NPM token configuration from the release workflow.

eng618 committed:
chore: Remove explicit npm registry URL configuration from Node.js setup in release workflow.

eng618 committed:
chore: Update release npm configuration to disable publishing and skip checks.

eng618 committed:
ci: Upgrade npm, enable npm provenance, and skip workspace checks for release-it.

eng618 committed:
fix: test release with empty commit

eng618 committed:
chore: modernize dev standards and refine release configuration

eng618 committed:
chore: configure release-it workspaces plugin to allow same version

eng618 committed:
chore: update dependencies across all packages, including Gatsby and Carbon components (#1674)

eng618 committed:
chore: Update Node.js engine to 20.x/22.x, configure release-it workspaces, and use `yarn install --immutable` in CI workflows.

eng618 committed:
fix(release): revert release-it config to v4.3.31 working state

eng618 committed:
ci: update npm configs

eng618 committed:
ci: update npm command

eng618 committed:
chore: release 4.3.32 [skip ci]

eng618 committed:
ci: update workspace config

eng618 committed:
chore: release 4.3.33 [skip ci]

eng618 committed:
fix(release): update publish command

eng618 committed:
chore: release 4.3.34 [skip ci]

eng618 committed:
fix(release): update publish command

eng618 committed:
fix(release): force release

eng618 committed:
chore: release 4.3.35 [skip ci]

dependabot[bot] committed:
build(deps): bump qs in the npm_and_yarn group across 1 directory (#1680)

dependabot[bot] committed:
build(deps): bump stylelint from 16.26.1 to 17.1.0 (#1677)

dependabot[bot] committed:
build(deps): bump focus-trap-react from 11.0.6 to 12.0.0 (#1679)

eng618 committed:
chore: release 4.3.36 [skip ci]

carbon-automation[bot] committed:
chore(release): update carbon deps (#1682)

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

eng618 committed:
chore: release 4.3.37 [skip ci]

dependabot[bot] committed:
build(deps): bump @isaacs/brace-expansion (#1681)

eng618 committed:
chore: release 4.3.38 [skip ci]

dependabot[bot] committed:
build(deps): bump the npm_and_yarn group across 1 directory with 2 updates (#1683)

dependabot[bot] committed:
build(deps): bump ajv in the npm_and_yarn group across 1 directory (#1684)

eng618 committed:
chore: release 4.3.39 [skip ci]

carbon-automation[bot] committed:
chore(release): update carbon deps (#1685)

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

eng618 committed:
chore: release 4.3.40 [skip ci]

dependabot[bot] committed:
build(deps): bump basic-ftp in the npm_and_yarn group across 1 directory (#1686)

dependabot[bot] committed:
build(deps): bump lerna from 9.0.3 to 9.0.5 (#1688)

dependabot[bot] committed:
build(deps): bump @commitlint/config-conventional from 20.4.0 to 20.4.2 (#1687)

eng618 committed:
chore: release 4.3.41 [skip ci]

dependabot[bot] committed:
build(deps): bump the npm_and_yarn group across 1 directory with 3 updates (#1690)

Bumps the npm_and_yarn group with 3 updates in the / directory: [minimatch](https://github.com/isaacs/minimatch), [multer](https://github.com/expressjs/multer) and [tar](https://github.com/isaacs/node-tar).


Updates `minimatch` from 3.1.2 to 3.1.4
- [Changelog](https://github.com/isaacs/minimatch/blob/main/changelog.md)
- [Commits](https://github.com/isaacs/minimatch/compare/v3.1.2...v3.1.4)

Updates `multer` from 2.0.2 to 2.1.0
- [Release notes](https://github.com/expressjs/multer/releases)
- [Changelog](https://github.com/expressjs/multer/blob/main/CHANGELOG.md)
- [Commits](https://github.com/expressjs/multer/compare/v2.0.2...v2.1.0)

Updates `tar` from 7.5.2 to 7.5.8
- [Release notes](https://github.com/isaacs/node-tar/releases)
- [Changelog](https://github.com/isaacs/node-tar/blob/main/CHANGELOG.md)
- [Commits](https://github.com/isaacs/node-tar/compare/v7.5.2...v7.5.8)

---
updated-dependencies:
- dependency-name: minimatch
  dependency-version: 3.1.4
  dependency-type: indirect
  dependency-group: npm_and_yarn
- dependency-name: multer
  dependency-version: 2.1.0
  dependency-type: indirect
  dependency-group: npm_and_yarn
- dependency-name: tar
  dependency-version: 7.5.8
  dependency-type: indirect
  dependency-group: npm_and_yarn
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump svgo in the npm_and_yarn group across 1 directory (#1691)

eng618 committed:
chore: release 4.3.42 [skip ci]

carbon-automation[bot] committed:
chore(release): update carbon deps (#1693)

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

eng618 committed:
chore: release 4.3.43 [skip ci]

carbon-automation[bot] committed:
chore(release): update carbon deps (#1694)

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

eng618 committed:
chore: release 4.3.44 [skip ci]

carbon-automation[bot] committed:
chore(release): update carbon deps (#1695)

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

tay1orjones committed:
fix(release): add team members to user check

sangeethababu9223 committed:
fix(actions): git actios version issue (#1698)

sangeethababu9223 committed:
fix(actions): regression work around (#1699)

* fix(actions): regression work around

* fix(node version): update to 22.21.1

* fix(action): revert dev node version

eng618 committed:
chore: release 4.3.45 [skip ci]

carbon-automation[bot] committed:
chore(release): update carbon deps (#1700)

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump actions/deploy-pages from 4 to 5 (#1696)

Bumps [actions/deploy-pages](https://github.com/actions/deploy-pages) from 4 to 5.
- [Release notes](https://github.com/actions/deploy-pages/releases)
- [Commits](https://github.com/actions/deploy-pages/compare/v4...v5)

---
updated-dependencies:
- dependency-name: actions/deploy-pages
  dependency-version: '5'
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump actions/configure-pages from 5 to 6 (#1697)

Bumps [actions/configure-pages](https://github.com/actions/configure-pages) from 5 to 6.
- [Release notes](https://github.com/actions/configure-pages/releases)
- [Commits](https://github.com/actions/configure-pages/compare/v5...v6)

---
updated-dependencies:
- dependency-name: actions/configure-pages
  dependency-version: '6'
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump multer in the npm_and_yarn group across 1 directory (#1692)

Bumps the npm_and_yarn group with 1 update in the / directory: [multer](https://github.com/expressjs/multer).


Updates `multer` from 2.1.0 to 2.1.1
- [Release notes](https://github.com/expressjs/multer/releases)
- [Changelog](https://github.com/expressjs/multer/blob/main/CHANGELOG.md)
- [Commits](https://github.com/expressjs/multer/compare/v2.1.0...v2.1.1)

---
updated-dependencies:
- dependency-name: multer
  dependency-version: 2.1.1
  dependency-type: indirect
  dependency-group: npm_and_yarn
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump stylelint-scss from 6.14.0 to 7.0.0 (#1689)

Bumps [stylelint-scss](https://github.com/stylelint-scss/stylelint-scss) from 6.14.0 to 7.0.0.
- [Release notes](https://github.com/stylelint-scss/stylelint-scss/releases)
- [Changelog](https://github.com/stylelint-scss/stylelint-scss/blob/master/CHANGELOG.md)
- [Commits](https://github.com/stylelint-scss/stylelint-scss/compare/v6.14.0...v7.0.0)

---
updated-dependencies:
- dependency-name: stylelint-scss
  dependency-version: 7.0.0
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

eng618 committed:
chore: release 4.3.46 [skip ci]

dependabot[bot] committed:
build(deps): bump actions/github-script from 8 to 9 (#1702)

dependabot[bot] committed:
build(deps): bump actions/upload-pages-artifact from 4 to 5 (#1701)

eng618 committed:
chore: release 4.3.47 [skip ci]

dependabot[bot] committed:
build(deps): bump the npm_and_yarn group across 1 directory with 10 updates (#1703)

dependabot[bot] committed:
build(deps): bump ip-address (#1704)

Bumps the npm_and_yarn group with 1 update in the / directory: [ip-address](https://github.com/beaugunderson/ip-address).


Updates `ip-address` from 10.1.0 to 10.2.0
- [Commits](https://github.com/beaugunderson/ip-address/commits)

---
updated-dependencies:
- dependency-name: ip-address
  dependency-version: 10.2.0
  dependency-type: indirect
  dependency-group: npm_and_yarn
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

eng618 committed:
chore: release 4.3.48 [skip ci]

dependabot[bot] committed:
build(deps): bump fast-uri in the npm_and_yarn group across 1 directory (#1705)

Bumps the npm_and_yarn group with 1 update in the / directory: [fast-uri](https://github.com/fastify/fast-uri).


Updates `fast-uri` from 3.1.0 to 3.1.2
- [Release notes](https://github.com/fastify/fast-uri/releases)
- [Commits](https://github.com/fastify/fast-uri/compare/v3.1.0...v3.1.2)

---
updated-dependencies:
- dependency-name: fast-uri
  dependency-version: 3.1.2
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump @babel/plugin-transform-modules-systemjs (#1706)

Bumps the npm_and_yarn group with 1 update in the / directory: [@babel/plugin-transform-modules-systemjs](https://github.com/babel/babel/tree/HEAD/packages/babel-plugin-transform-modules-systemjs).


Updates `@babel/plugin-transform-modules-systemjs` from 7.28.5 to 7.29.4
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/main/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/commits/v7.29.4/packages/babel-plugin-transform-modules-systemjs)

---
updated-dependencies:
- dependency-name: "@babel/plugin-transform-modules-systemjs"
  dependency-version: 7.29.4
  dependency-type: indirect
  dependency-group: npm_and_yarn
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

eng618 committed:
chore: release 4.3.49 [skip ci]

carbon-automation[bot] committed:
chore(release): update carbon deps (#1710)

Co-authored-by: maradwan26 <132728978+maradwan26@users.noreply.github.com>

eng618 committed:
chore: release 4.3.50 [skip ci]
